src/vmm/src/cpu_config/templates.rs

```diff
@@ -193,6 +213,13 @@ where
         let (mut filter, mut value) = (V::zero(), V::zero());
         let mut i = 0;
         for s in stripped_str.as_bytes().iter().rev() {
+            if V::BITS == i {
+                return Err(D::Error::custom(format!(
+                    "Failed to parse string [{}] as a bitmap - string is too long",
+                    original_str
+                )));
```

Comment:
Can this be hoisted out of the loop? like if `V::BITS < stripped_str.as_bytes().len() - 1 { return ... }`

Reply:
Underscores should not be count as a part of the string, so we can not make this check before the loop.

Reply:
Ah, missed the underscore thing. Sorry!